### ISSUE TITLE
yolo9000

### ISSUE THREAD

**Issue by bjornamr** [allanzelener/YAD2K#25]
Does this work for yolo9000? I cant make it work right now

**Comment by allanzelener** [comment]
No, the softmax tree method described by YOLO9000 is not currently implemented here. Most of the other YOLO_v2 features are.

**Comment by ryubidragonfire** [comment]
Has there been any update for yolo9000?  is there an example on how to:
1. use yolo9000 in Keras
2. how to re-train with own data for different number of classes

**Comment by zeevikal** [comment]
any news?

**Comment by average-everyman** [comment]
any news? is yolo9000 supported now?
